parent/pom.xml

```diff
@@ -183,6 +183,7 @@
                     <version>2.1.0</version>
                     <executions>
                         <execution>
+                            <id>install-bundle</id>
```

Comment:
shouldn't it rather be named `autoInstallBundle` to match the naming pattern of `autoInstallSinglePackage`?
Also it should be clarified how those two should be used together.

Reply:
That's not a profile name, it's just an execution id.

Reply:
Sorry, makes sense then!